migrations/1669984610620-DepositsMv.ts

```diff
@@ -0,0 +1,81 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class DepositsMv1669984610620 implements MigrationInterface {
+  name = "DepositsMv1669984610620";
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`DROP MATERIALIZED VIEW IF EXISTS "deposits_mv"`);
+    await queryRunner.query(`DROP VIEW IF EXISTS "deposits_filtered_referrals"`);
+    await queryRunner.query(`CREATE VIEW "deposits_filtered_referrals" AS 
+        SELECT 
+        d.id,
+        d."depositorAddr",
+        d."stickyReferralAddress",
+        d."depositDate",
+        d."priceId",
+        d."tokenId",
+        d.amount,
+        d."rewardsWindowIndex",
+        case when d."rewardsWindowIndex" = c."windowIndex" then d."rewardsWindowIndex" else -1 end as "referralClaimedWindowIndex",
+        hmp."usd",
+        t.decimals
+      FROM deposit d
+      JOIN historic_market_price hmp on d."priceId" = hmp.id
+      JOIN token t on d."tokenId" = t.id
+      LEFT JOIN claim c on d."rewardsWindowIndex" = c."windowIndex" and d."stickyReferralAddress" = c."account"
+      WHERE "stickyReferralAddress" is not null
+        AND "depositDate" is not null
+        AND "tokenId" is not null
+        AND "priceId" is not null
+        AND status = 'filled'
+        AND d."acxUsdPrice" is not null;    
+    `);
+
+    await queryRunner.query(`CREATE MATERIALIZED VIEW "deposits_mv" AS 
+      SELECT
+        d.id,
+        d."depositId",
+        d."depositTxHash",
+        d."sourceChainId",
+        d."destinationChainId",
+        d.amount,
+        t.symbol,
+        t.decimals,
+        d."depositorAddr",
+        d."rewardsWindowIndex",
+        case when d."rewardsWindowIndex" = c."windowIndex" and d."depositorAddr" = c.account then d."rewardsWindowIndex" else -1 end as "depositorClaimedWindowIndex",
+        d1."referralClaimedWindowIndex",
+        d."stickyReferralAddress" AS "referralAddress",
+        d."depositDate",
+        hmp.usd AS "tokenUsdPrice",
+        (d."realizedLpFeePctCapped" / power(10, 18)) * (d.amount / power(10, t.decimals)) * hmp.usd AS "realizedLpFeeUsd",
+        (d."bridgeFeePct" / power(10, 18)) * (d.amount / power(10, t.decimals)) * hmp.usd AS "bridgeFeeUsd",
+        d."acxUsdPrice",
```

Comment:
This is the only line that changed in the `deposits_mv` MATERIALIZED VIEW definition. THe `acxUsdPrice` needs to be exposed because the referral rewards amounts are computed based on it instead of using the hardcoded 0.1 value.